pkg/utils/utils_test.go

```diff
@@ -7,11 +7,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var loc = time.Now().Local().Location()
+// Utiliser UTC au lieu de Local pour éviter les problèmes de fuseaux horaires
+var loc = time.UTC
```

Comment:
a future datetime should not use UTC, any change in TZ rules could put it out of sync.
the previous usage of `.Local()` means we can just set the correct TZ env and get the behavior we expect. So with that in mind is there really anything to fix and can the refactor to the "multi-day" event feature be done another way ?